manifests/params.pp

```diff
@@ -12,6 +12,7 @@
   $scala_version  = '2.11'
   $install_dir    = "/opt/kafka-${scala_version}-${version}"
   $config_dir     = '/opt/kafka/config'
+  $bin_dir        = '/opt/kafka/bin'
```

Comment:
Since we're on puppet4 (or even 5 \o/) now, can you please use the datatype Stdlib::Absolutepath here (and in the other classes where you use the $bin_dir)?